docs/dawson/usecase.md

```diff
@@ -1,89 +1,323 @@
+# ADCIRC Use Case - Using Tapis and Pylauncher for Ensemble Modeling in DesignSafe 
 
-**Authors, Affiliations**  
+**Clint Dawson, University of Texas at Austin**  
+**Carlos del-Castillo-Negrete, University of Texas at Austin**  
+**Benjamin Pachev, University of Texas at Austin**  
 
-Introductory Text.  High-level overview of product. A condimentum vitae sapien pellentesque habitant morbi tristique.
+The following use case presents an example of how to leverage the Tapis API to run an ensemble of HPC simulations. The specific workflow to be presented consists of running ADCIRC, a storm-surge modeling application available on DesignSafe, using the parametric job launcher pylauncher. All code and examples presented are meant to be be executed from a Jupyter Notebook on the DesignSafe platform and using a DesignSafe account to make Tapis API calls. Accompanying notebooks for this use case can be found in the ADCIRC folder in [Community Data](https://www.designsafe-ci.org/data/browser/public/designsafe.storage.community/Use%20Case%20Products).
+
+Learn more: [Jupyter notebooks on DS Juypterhub](https://www.designsafe-ci.org/rw/workspace/#!/Jupyter::Analysis).
 
 ## Background 
 
 ### Citation and Licensing
 
-* Please cite [AUTHORS et al. (20xx) - example of published project](https://doi.org/10.17603/ds2-3zdj-493) to acknowledge the use of any resources from this use case.
+* Please cite [Dawson et al. (2021)](https://doi.org/10.17603/ds2-68a9-0s64) if using any of the Texas FEMA Hurricane data used in the examples below. 
 
 * Please cite [Rathje et al. (2017)](https://doi.org/10.1061/(ASCE)NH.1527-6996.0000246) to acknowledge the use of DesignSafe resources.  
 
 * This software is distributed under the GNU General Public License (https://www.gnu.org/licenses/gpl-3.0.html).  
 
-### Description 
+### ADCIRC
+
+For more information on running ADCIRC and documentation, see the following links:
+
+* [ADCIRC Wiki](https://wiki.adcirc.org/wiki/Main_Page)
+* [ADCIRC Web Page](https://adcirc.org/)
+
+ADCIRC is available as a standalone app accesible via the [DesignSafe front-end](https://www.designsafe-ci.org/rw/workspace/#!/ADCIRC::Simulation).
+
+ADCIRC files used in this demo are pre-staged on TACC resources that DesignSafe execution systems have access to, at the path `/work/06307/clos21/pub/adcirc`. See the section on using data from Projects directory for using other data sources.
+
+### Tapis
+
+Tapis is the main API to control and access HPC resources with. For more resources and tutorials on how to use Tapis, see the following:
+
+- [Tapis CLI](https://tapis-cli.readthedocs.io/en/latest/contents.html)
+* [AgavePy](https://tacc-cloud.readthedocs.io/projects/agave/en/latest/)
+* [DesignSafe Webinar](https://www.youtube.com/watch?v=-_1lNWW8CAg&t=1854s&ab_channel=NHERIDesignSafe-CIMedia)
+
+To initialize tapis in our jupyter notebook we use AgavePy. Relies on `tapis auth init --interactive` being run from CLI first.
```

Comment:
Could you please clarification on where this should be run with some screenshots?

Reply:
I've added an image of executing the `tapis auth init --interactive` call from within a terminal in jupyter.